bayesian/v2/utility.py

```diff
@@ -0,0 +1,348 @@
+# Copyright © 2020 Red Hat Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# Author: Deepak Sharma <deepshar@redhat.com>
+#
+"""Utility function from API v2."""
+
+from urllib.parse import quote
+import logging
+from bayesian.v2.communicator import GraphAnalyses
+import semantic_version as sv
+
+
+logger = logging.getLogger(__file__)
+
+
+class Adequacy:
```

Comment:
Why do we need a class with only static methods.

Reply:
@prashbnair Class is added to maintain Uniformily. It will help in following OOP if required in future.

Reply:
As I see it, this is only for namespacing now and this can be achieved using modules. When future requirements come in we can move it into a class. A class needs to have state and instance methods both of which are not present here.

Reply:
Ack! Removed class Adequacy, Converted all method to standalone